Codeinwp/neve
# feat: add line social icon to icon selector

### Summary

Supporting changes to allow Line social icon inside Social Icon component.

### Will affect the visual aspect of the product

NO

### Screenshots 
![image](https://user-images.githubusercontent.com/23024731/197779367-dc6f371f-ab29-49d5-93da-38bb97b50e44.png)

### Test instructions

1. Check that you can use and display the new icon


Closes Codeinwp/neve-pro-addon#2093.
